### PR TITLE
feat: add an option to replace the sample input/output

### PR DESCRIPTION
**Pull request by cubercsl**

### CODE REVIEW

File: p2d/p2d.py [hunk 1]
```diff
@@ -206,10 +207,10 @@ def compare(src: Path, dst: Path):
                 # interactor can not support custom sample because DOMjudge always use sample input to test
                 sample_input_src = self.package_dir / 'statements' / self.language / (sample_input_path_pattern % idx)
                 sample_output_src = self.package_dir / 'statements' / self.language / (sample_output_path_pattern % idx)
-                if sample_input_src.exists():
+                if self.replace_sample and sample_input_src.exists():
                     compare(input_src, sample_input_src)
                     input_src = sample_input_src
-                if sample_output_src.exists():
+                if self.replce_sample and sample_output_src.exists():
```
Comment (Dup4) [inline p2d/p2d.py]: image

笑死了，本地不跑的？

Comment (Dup4) [reply p2d/p2d.py]: 怎么 spell checker 也不装的

https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Comment (cubercsl) [reply p2d/p2d.py]: 草

**Review by Dup4 — APPROVED** [review]
LGTM

**Pull request merged**